### PR TITLE
Update ru_ru translation and fix en_US typo

### PR DESCRIPTION
**Pull request by ppv1979**

### CODE REVIEW

File: plugins/generic/acron/locale/ru_RU/locale.xml [hunk 1]
```diff
@@ -2,9 +2,9 @@
 <!DOCTYPE locale SYSTEM "../../../../../dtd/locale.dtd">
 
 <!--
-  * plugins/generic/acron/locale/ru_RU/locale.xml
+  * locale.xml
   *
-  * Copyright (c) 2014-2016 Simon Fraser University Library
+  * Copyright (c) 2013-2016 Simon Fraser University Library
```
Comment (mtub) [inline plugins/generic/acron/locale/ru_RU/locale.xml]: good spotting...

File: locale/ru_RU/admin.xml [hunk 2]
```diff
@@ -9,79 +9,126 @@
   * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
   *
   * Localization strings for the ru_RU (Русский) locale.
-  *
+  * With updates from Pavel Pisklakov (revising translation).
```
Comment (ppv1979) [inline locale/ru_RU/admin.xml]: Yes, you can replace it with this link.

File: locale/ru_RU/common.xml [hunk 3]
```diff
@@ -9,339 +9,540 @@
   * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
   *
   * Localization strings for the ru_RU (Русский) locale.
-  *
+  * With updates from Pavel Pisklakov (revising translation).
```
Comment (ppv1979) [inline locale/ru_RU/common.xml]: Yes, you can replace it with this link.

File: locale/ru_RU/currencies.xml [hunk 4]
```diff
@@ -8,177 +8,170 @@
   * Copyright (c) 2000-2016 John Willinsky
   * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
   *
-  * Localized list of currencies.
-  *
+  * Currencies data XML file. Based on ISO 4217.
```
Comment (ppv1979) [inline locale/ru_RU/currencies.xml]: I've translated the file from en_US locale and checked it with some Russian sources.

File: locale/ru_RU/languages.xml [hunk 5]
```diff
@@ -0,0 +1,201 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE languages SYSTEM "../../dtd/languages.dtd">
+<!--
+  * locale/ru_RU/languages.xml
+  *
+  * Copyright (c) 2014-2016 Simon Fraser University Library
+  * Copyright (c) 2000-2016 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * Localized list of common languages. Based on ISO 639-1.
```
Comment (ppv1979) [inline locale/ru_RU/languages.xml]: I've translated the file from en_US locale and checked it with some Russian sources.

File: plugins/generic/acron/locale/ru_RU/locale.xml (hunk 1 above)
Comment (ppv1979) [inline plugins/generic/acron/locale/ru_RU/locale.xml]: Ok.

**Pull request merged**